Lib/test/test_compile.py

```diff
@@ -2204,17 +2204,17 @@ def f():
             start_line, end_line, _, _ = instr.positions
             self.assertEqual(start_line, end_line)
 
-        # Expect three load None instructions for the no-exception __exit__ call,
-        # and one RETURN_VALUE.
+        # Expect three `LOAD_CONST None` instructions for the no-exception __exit__ call,
+        # and one for the RETURN_VALUE.
```

Comment:
I think this should be 4 and 2 based on the asserts below.
```suggestion
        # Expect four `LOAD_CONST None` instructions for the no-exception __exit__ call,
        # and two for the RETURN_VALUE.
```

Reply:
three for for the __exit__ call and one for the return is four.
I'll clarify.